src/AcceptanceTests/SFA.DAS.Funding.ApprenticeshipEarnings.AcceptanceTests.csproj

```diff
@@ -64,30 +64,11 @@
   </ItemGroup>
 
   <ItemGroup>
-    <Compile Update="Features\RecalculateEarningsFollowingWithdrawalBeforeEndOfQualifyingPeriod.feature.cs">
-      <Visible>true</Visible>
-      <DependentUpon>RecalculateEarningsFollowingWithdrawalBeforeEndOfQualifyingPeriod.feature</DependentUpon>
-      <DesignTime>true</DesignTime>
-      <AutoGen>true</AutoGen>
-    </Compile>
-    <Compile Update="Features\RecalculateEarningsFollowingWithdrawal.feature.cs">
-      <DependentUpon>RecalculateEarningsFollowingWithdrawal.feature</DependentUpon>
-    </Compile>
-    <Compile Update="Features\RecalculateEarningsFollowingStartDateChange.feature.cs">
-      <DependentUpon>RecalculateEarningsFollowingStartDateChange.feature</DependentUpon>
-    </Compile>
-  </ItemGroup>
-
-  <ItemGroup>
-    <SpecFlowFeatureFiles Update="Features\RecalculateEarningsFollowingWithdrawalBeforeEndOfQualifyingPeriod.feature">
-      <Visible>$(UsingMicrosoftNETSdk)</Visible>
-      <CodeBehindFile>%(RelativeDir)%(Filename).feature$(DefaultLanguageSourceExtension)</CodeBehindFile>
-    </SpecFlowFeatureFiles>
-    <SpecFlowFeatureFiles Update="Features\RecalculateEarningsFollowingWithdrawal.feature">
+    <SpecFlowFeatureFiles Update="Features\UpdateOnProgramme\Withdrawal.feature">
       <Visible>$(UsingMicrosoftNETSdk)</Visible>
       <CodeBehindFile>%(RelativeDir)%(Filename).feature$(DefaultLanguageSourceExtension)</CodeBehindFile>
     </SpecFlowFeatureFiles>
-    <SpecFlowFeatureFiles Update="Features\RecalculateEarningsFollowingStartDateChange.feature">
+    <SpecFlowFeatureFiles Update="Features\UpdateOnProgramme\StartDateChange.feature">
       <Visible>$(UsingMicrosoftNETSdk)</Visible>
       <CodeBehindFile>%(RelativeDir)%(Filename).feature$(DefaultLanguageSourceExtension)</CodeBehindFile>
     </SpecFlowFeatureFiles>
```

Comment:
Do we need these explicit specflow lines? I ask because it seems like we only have 2 of them for 2 of the files so perhaps they are not necessary at all